mycity/mycity/intents/trash_intent.py

```diff
@@ -153,23 +153,19 @@ def get_trash_and_recycling_days(address, neighborhood=None):
     return trash_and_recycling_days
 
 
-def find_unique_addresses(address_request_json):
+def find_unique_address(address_request_json):
     """
-    Finds unique addresses in a provided address request json returned
+    Finds the first address in a provided address request json returned
     from the ReCollect service
     :param address_request_json: json object returned from ReCollect address
         request service
-    :return: list of unique addresses
+    :return: list of one unique address
     """
     logger.debug('address_request_json: ' + str(address_request_json))
     # Pre-extract the addresses from the payload and uniquify them
-    strings_to_compare = sorted(set(address["name"] for address in address_request_json), key=len, reverse=True)
+    # strings_to_compare = sorted(set(address["name"] for address in address_request_json), key=len, reverse=True)
+    return [address_request_json[0]["name"]]
```

Comment:
I can add documentation why we're using the first address here

Reply:
That'd be great.

Reply:
Thanks for the help. Happy new year